src/main/java/org/junitpioneer/jupiter/ByteRangeSource.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2015-2018 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.junit.jupiter.params.provider.ArgumentsSource;
+
+/**
+ * {@code @ByteRangeSource} is an {@link ArgumentsSource} which provides access to a range of {@code byte} values.
+ *
+ * <p>The supplied values will be provided as arguments to the annotated {@code @ParameterizedTest} method.
+ *
+ * @since 4.0
+ * @see ArgumentsSource
+ * @see org.junit.jupiter.params.ParameterizedTest
+ */
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@ArgumentsSource(RangeSourceProvider.class)
+public @interface ByteRangeSource {
```

Comment:
Hm, if we want to use this source for #68, we need this source to be repeatable. No reason to implement that now, though (I guess?).